app/helpers/mustache_helper.rb

```diff
@@ -52,7 +52,8 @@ def js_version
 
   def js_variables
     'var js_path="' + styleguide_path('/js/dist/') + '"; ' +
-    'var require = {"urlArgs": "' + js_version  + '"};'
+    'var require = {"urlArgs": "' + js_version  + '"}; ' +
```

Comment:
Unnecessary spacing detected.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.